_includes/sites-world-map.html

```diff
@@ -0,0 +1,134 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet" href="https://openlayers.org/en/v5.3.0/css/ol.css" type="text/css">
+  <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=requestAnimationFrame,Element.prototype.classList,URL"></script>
+
+  <style>
+    .map {
+      height: 100%;
+      width: 100%;
+    }
+  </style>
+
+  <!-- Grab whole OpenLayers package from source -->
+  <script src="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/build/ol.js"></script>
+  <title>Cylc Usage Across the World</title>
+</head>
+
+<body>
+
+  <div id="map" class="map"></div>
+  <script type="text/javascript">
+    // Lat-lon decimal degrees coordinates (to ideally 5 s.f.) for major...
+    // ... known Cylc-using sites, with keys as neat names for labelling:
+    var cylcSiteLatLonCoors = {
+      "Met Office": [50.72762, -3.47564],
+      "Centre for Environmental Data Analysis (CEDA)": [51.57488, -1.31451],
+      "Plymouth Marine Laboratory": [50.36615, -4.14782],
+      "National Centre for Atmospheric Science (NCAS)": [53.80450, -1.56147],
+      "National Institute of Water & Atmospheric Research (NIWA)": [-36.8731, 174.7773],
+      "Max-Planck-Institut für Meteorologie": [53.56772, 9.97524],
+      "Deutches Klimarechenzentrum (DKRZ)": [53.56686, 9.97658],
+      "Bureau of Meteorology": [-37.81966, 144.95086],
+      "Meteorological Service Singapore (MSS)": [1.35380, 103.98875],
+      "NRL Marine Meteorology Division": [36.59290, -121.85459],
+      "Geophysical Fluid Dynamics Laboratory (GFDL)": [40.34539, -74.61623],
+      "Phillippine Atmospheric, Geophysical, & Astronomical Services Administration (PAGASA)": [14.64178, 121.04441],
+      "South African Weather Service (SAWS)": [-25.89068, 28.17072],
+      "National Centre for Medium Range Weather Forecasting (NCMRWF)": [28.62447, 77.35917],
+      "Korean Meteorological Administration (KMA)": [37.494, 126.917],
+      "Centre of Excellence in Simulation of Weather and Climate in Europe (ESiWACE)": [53.56686, 9.97658],
+      "NOAA Center for Weather and Climate Prediction (NCWCP)": [38.972, -76.925],
+      "Euro-Mediterranean Center on Climate Change (CMCC)": [40.35225, 18.17165],
+      "Barcelona Supercomputing Center (BSC)": [41.389, 2.116],
+      "National Center for Atmospheric Reserach (NCAR)": [39.97788, -105.27486],
```

Comment:
s/Reserach/Research

Reply:
Old typo too

Reply:
Yes, I didn't change any text I just transposed it for the nested list. But good spot, I'll amend any typos found now.